### PR TITLE
support view options

### PR DESCRIPTION
**Pull request by yyscamper**
The syntax for Postgresql: https://www.postgresql.org/docs/11/sql-createview.html
The usage on pipelinedb: http://docs.pipelinedb.com/quickstart.html

### CODE REVIEW

File: sqlalchemy_views/views.py [hunk 1]
```diff
@@ -49,6 +53,11 @@ def visit_create_view(create, compiler, **kw):
         text += "("
         text += ', '.join(column_names)
         text += ") "
+    if create.options:
+      ops = []
+      for opname, opval in create.options.items():
+        ops.append('='.join([str(opname), str(opval)]))
+      text += 'WITH (%s) ' % (','.join(ops))
```
Comment (jklukas) [inline sqlalchemy_views/views.py:60]:
```suggestion
      text += 'WITH (%s) ' % (', '.join(ops))
```

**Review by jklukas — COMMENTED** [review]
This looks great. Just one spacing suggestion for better readability in the output.

**Comment by bochecha** [comment]
This seems very important. Is it really only blocked on the spacing issue?

In other projects, the maintainer will often fix such issues themselves in a subsequent commit (or directly updating the offending commits) to avoid asking too much of the submitter.

In any case, I'm happy to send a fixed merge request (preserving @yyscamper's credit obviously) if that can unblock this. :slightly_smiling_face:

**Comment by jklukas** [comment]
> This seems very important. Is it really only blocked on the spacing issue?
> 
> In other projects, the maintainer will often fix such issues themselves in a subsequent commit (or directly updating the offending commits) to avoid asking too much of the submitter.

This is mostly a case of the maintainers (me) having low enough bandwidth that we aren't actively scanning through old PRs. So if I ask for a change and don't hear anything more, it's likely to completely fall off the radar.

But I think you're right in this case that merging and fixing up is reasonable. I will do so now.

**Comment by jklukas** [comment]
This is now published as 0.2.4: https://pypi.org/project/sqlalchemy-views/

**Comment by bochecha** [comment]
Woah, and with a new release even. Thank you! :heart:

**Pull request merged**